adapters/bmtm/brightmountainmedia.go

```diff
@@ -0,0 +1,160 @@
+package bmtm
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"github.com/mxmCherry/openrtb/v15/openrtb2"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+type BrightMountainMediaAdapter struct {
```

Comment:
We prefer to use `adapter` as the name for this struct. There is no need to repeat the package name. There is no need to export the type using an uppercase letter.

Reply:
Please have a look @SyntaxNode